hack/jenkins/e2e.sh

```diff
@@ -219,7 +224,10 @@ fi
 # Jenkins will look at the junit*.xml files for test failures, so don't exit
 # with a nonzero error code if it was only tests that failed.
 if [[ "${E2E_TEST,,}" == "true" ]]; then
-    go run ./hack/e2e.go ${E2E_OPT} -v --test --test_args="${GINKGO_TEST_ARGS}--ginkgo.noColor" || true
+    # GINKGO_ARGS is parsed by the Ginkgo runner itself, whereas the --test_args
```

Comment:
The flag description is actually wrong. The values in test_args is passed as arguments to hack/ginkgo-e2e.sh, which just tacks them on to the end of the ginkgo command, along with the rest of the flags passed to the test (rather than the ginkgo runner).

I was trying for just a quick fix, but I guess I should maybe make a slightly larger change to clean all of this up a bit?